Userland/Utilities/man.cpp

```diff
@@ -91,13 +91,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 break;
             }
         }
-        if (!section) {
```

Comment:
Should this not be kept here? The way I'm reading the code, if I type `man 2 testing` but only `man 3 testing`, previously, we would properly fail. Now, we'll succeed. Is this correct?

Reply:
The problem was that the check if the path can be resolved was only done if no section was specified on the command line.
Because of this, this happens:
```
$ man 2 testing
Runtime error: No such file or directory (errno=2)
ASSERTION FAILED: i < m_size
```
etc. (we fell through, even though the manpage could not be resolved)
While
```
$ man testing
```
properly fails with
```
No man page for testing
```

After the fix, the desirable happens:
```
$ man 2 testing
No man page for testing
```

The check still happens if no section was specified, only afterwards, and for both paths.

Reply:
Gotcha, thank you! Mark this as resolved so that the maintainers can see that this is not an issue.